### PR TITLE
Add EinfachGustaf.live Minecraft server

### PR DESCRIPTION
**Pull request by DinoMarlir**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/einfachgustaf/background.png
Comment (maaaathis) [inline minecraft_servers/einfachgustaf/background.png:1]: Compress this file

File: minecraft_servers/einfachgustaf/manifest.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "server_name": "einfachgustaf",
+    "nice_name": "EinfachGustaf.live",
+    "direct_ip": "einfachgustaf.live",
+    "server_wildcards": [
+      "%.einfachgustaf.live"
```
Comment (maaaathis) [inline minecraft_servers/einfachgustaf/manifest.json:6]: Doesn't look like a real wildcard.
image

File: minecraft_servers/einfachgustaf/manifest.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ],
+    "social": {
+      "web": "https://einfachgustaf.live",
+      "twitter": "einfachgustafl",
+      "discord": "https://discord.gg/R5spCd5vq8",
+      "youtube": "https://www.youtube.com/@einfachgustaflive"
+    },
+    "gamemodes": {
+      "jumpworld": {
+        "name": "SMP",
```
Comment (maaaathis) [inline minecraft_servers/einfachgustaf/manifest.json:19]: Key doesn't match to name

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by DinoMarlir** [comment]
Thank you for reviewing. Everything is fixed 👍

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**